platforms/quorum/configuration/roles/create/k8_component/templates/eks_storageclass.tpl

```diff
@@ -0,0 +1,10 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: {{ component_name }}
+provisioner: kubernetes.io/aws-ebs
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+parameters:
+  encrypted: "true" 
+   
```

Comment:
This is not right, the red mark there should not be there if you did it correctly. For all the files.